### ISSUE TITLE
Consider deprecating https://docs.openobservatory.net/ooni/probe/

### ISSUE THREAD

**Issue by ainghazal** [ooni/probe#2395]
this looks like documentation for the legacy python codebase; it probably should be deprecated?

**Comment by hellais** [comment]
Yes this should be deprecated. I didn't even know it existed. I think it's probably a CNAME to readthedocs.
